.adops/package-update.yml

```diff
@@ -87,15 +98,32 @@ steps:
 
   - script: |
       git fetch
-      git checkout $(buildSourceBranch)
-      git pull origin $(buildSourceBranch)
       git add .
       git commit -m "chore: bumped Mosaic packages [skip ci]"
-      git push origin $(buildSourceBranch)
+      
+      echo "Pushing to temporary branch: $(temporaryPushBranch) to evaluate successful status check"
+      git checkout -b $(temporaryPushBranch)
+      git push origin $(temporaryPushBranch)
```

Comment:
is pushing to remote enough to trigger a check-run? was looking for some documentation on this, but I didn't find something that clearly says this